infrastructure/zk/src/docker.ts

```diff
@@ -67,19 +67,40 @@ export async function push(image: string, tag: string) {
 }
 
 export async function pushFromTag(gitTag: string) {
-    const [image_name, image_tag] = gitTag.split(':');
-    await dockerCommand('build', image_name, image_tag);
-    await dockerCommand('push', image_name, image_tag);
+    const { imageName, imageTag } = getNameTag(gitTag);
+    await dockerCommand('build', imageName, imageTag);
+    await dockerCommand('push', imageName, imageTag);
 }
 
 export async function restart(container: string) {
     await utils.spawn(`docker-compose restart ${container}`);
 }
 
+export async function deployUp(service: string) {
+    console.log('WARNING! Using docker!');
+    await utils.spawn(`docker-compose -f docker-compose.deploy.yml up ${service}`);
+}
+
+export async function deployRun(command: string) {
+    console.log('WARNING! Using docker!');
+    await utils.spawn(`docker-compose -f docker-compose.deploy.yml run ${command}`);
+}
```

Comment:
NIT (sorry) 😅: we could define `console.log('WARNING! Using docker!');` just once.

Reply:
updated